i62 Collection splash page

kirkkwang opened the pull request:
# Story

This PR will add more styling to the Collection splash page.  Most
notable is that the banner image is much bigger and more like the DC
collection page.

Ref:
- https://github.com/scientist-softserv/utk-hyku/issues/62

# Expected Behavior Before Changes
- Banner was too small

# Expected Behavior After Changes
- Banner is much bigger
- Other small tweaks


# Screenshots / Video

<details>
<summary>Before</summary>

![image](https://github.com/scientist-softserv/utk-hyku/assets/19597776/41e6ca9f-93a8-476a-8d56-d27861fa2ea7)

</details>

<details>
<summary>After</summary>

![image](https://github.com/scientist-softserv/utk-hyku/assets/19597776/1adc96cd-e902-47bc-89be-5b0b9a131fec)

</details>

DeonFranklin committed:
Collection splash page updates

The changes in this commit will restyle the collection splash page
in accordance with the approved figma design.

DeonFranklin committed:
Hide total items badge on the collection show page

Commit:
🎁 Modify styling of the Collection splash page

This commit will add more styling to the Collection splash page.  Most
notable is that the banner image is much bigger and more like the DC
collection page.

Ref:
  - https://github.com/scientist-softserv/utk-hyku/issues/62

Commit:
🎁 Make items and last updated one line

This commit will move the `items` and `last updated` information to one
line.  Also I am adding the branding folder to the `.gitignore` file so
when we upload new images in we won't see a diff.

jeremyf commented on app/views/catalog/_index_default.html.erb:
```diff
@@ -3,18 +3,6 @@
 <% doc_presenter = index_presenter(document) %>
 <%# default partial to display solr document fields in catalog index view %>
 <% if search_results_theme == 'dc_repository_search' && home_page_theme == 'dc_repository' %>
-  <div class="document-metadata">
```
Could you update the override comment to include why you're removing this (maybe a link to the issue?)

kirkkwang replied:
Sounds good, I'll try and track this one down, this was Deon's commit so off the top of my head I'm not sure exactly what's going on yet.

kirkkwang replied:
I didn't find anything that supported removing this element, I've brought it back

Commit:
🎁 Bring back part of collection

This commit will bring back the "Part of <collection>" text on the
search results page.  I couldn't find any paper trail of if this was
to be removed.  Also, I removed the nav breadcrumbs from the collections
show page because it was requested in the Figma design.

Commit:
Merge branch 'main' into collection-splash-page

Commit:
🎁 Remove subpage header for collections show

This commit will remove the subpage header (the bottom one) for the
collections show page.  I added a helper method to keep things tidy.
